source/default/_patterns/02-molecules/carousel/index.ts

```diff
@@ -21,14 +21,18 @@ export const defaults = {
   interval: 3000,
```

Comment:
I can't comment above this line, but shouldn't this be cast/typed to `CarouselSettings`?

Reply:
Doesn't need to be. TypeScript recognizes that it is of the same data shape as the interface declared above and infers it. It can be added, but it is an unnecessary annotation as far as the compiler is concerned.

Reply:
Going to move it above the export so it is clearer as well.

Reply:
So with the interface below this point, TS is cool with this shape from here on?

Reply:
At least within this file. If it were to be needed in another file it would be exported from here and then imported (similar to what is being done in Vue-clock).